distributed/tests/test_client.py

```diff
@@ -5498,5 +5499,33 @@ def test_profile_bokeh(c, s, a, b):
         assert os.path.exists(fn)
 
 
+@gen_cluster(client=True)
+def test_get_mix_futures_and_SubgraphCallable(c, s, a, b):
```

Comment:
It would be useful to also have another test with user-facing API.  I wouldn't be surprised if, in the future, we refactor the `SubgraphCallable` into something else.  Having a test with the user-facing API would probably have more sticking power.

Reply:
Done